server/concepts/Tournament.ts

```diff
@@ -123,13 +124,64 @@ export default class Tournament implements TournamentWithoutMS {
     /* eslint-enable @typescript-eslint/no-non-null-assertion */
   }
 
-  public static async getById(tournamenId: string, token: string) {
-    const tournament = await TournamentRepository.getById(tournamenId);
+  public static async getById(tournamentId: string, token: string) {
+    const tournament = await TournamentRepository.getById(tournamentId);
     if (!tournament) return null;
     const enrichedTournament = await Tournament.enrichWithMSUsers(tournament, token);
+    // const teams = this.countPointsPerTeam(enrichedTournament);
+    // console.log(teams);
     return enrichedTournament;
   }
 
+  public static countPointsPerTeam(enrichedTournament: ITournament) {
+    // teams: [
+    //   {
+    //     name: "Team 1",
+    //     wins: "1",
+    //     draws: '0',
+    //     loses: '1',
+    //     points: '3',
+    //     matchesPlayed?
+    //   },
+    //   {
+    //     ...
+    //   }
+    // ]
+    const teams: IPointsPerTeam[] = [];
+    enrichedTournament.teams.forEach((team: ITeam) => {
+      let points = 0;
+      let draws = 0;
+      let loses = 0;
+      let wins = 0;
+
+      // eslint-disable-next-line no-restricted-syntax
+      for (const match of enrichedTournament.matches) {
+        if (match.score.final.a === -1 || match.score.final.b === -1) continue;
+        // eslint-disable-next-line no-nested-ternary
+        const teamIdentifierLetter = team.id === match.teamA?.id ? 'a' : (team.id === match.teamB?.id ? 'b' : undefined);
+        if (!teamIdentifierLetter) continue;
+        if (match.score.final[teamIdentifierLetter] > match.score.final[teamIdentifierLetter === 'a' ? 'b' : 'a']) {
```

Comment:
```suggestion
        const opositeTeamIdentifierLetter = teamIdentifierLetter === 'a' ? 'b' : 'a';
        if (match.score.final[teamIdentifierLetter] > match.score.final[opositeTeamIdentifierLetter]) {
```
I'm not sure that, but maybe move `opositeTeamLetter` to variable instead of calculating this every time. Due to the visual side of the code. 
Second option might be to move left and right sides of conditions to two variables (e.q. `teamScore`, `opositeTeamScore`) and compare them

Reply:
I can add `oppositeTeamLetter`